x-pack/packages/kbn-entities-schema/oas.yaml

```diff
@@ -8,7 +8,12 @@ components:
       type: object
       properties:
         installOnly:
-          type: boolean
+          anyOf:
```

Comment:
I can see this being defined as Boolean only in other routes, do we know why ?

Reply:
So it comes from the type `createEntityDefinitionQuerySchema` where it's defined as `BooleanFromString`, and we then link that type into the schema as a component (that's the definition you're looking at here).

But within the routes, we pass `createEntityDefinitionQuerySchema` as part of another schema so I think the link between them is lost and then it's inferred down to just boolean because that's what `BooleanFromString` outputs.

I guess to fix this, we would need to somehow communicate that the `createEntityDefinitionQuerySchema` schema is being used as a subschema, no idea how one would do that.